processor/src/test/java/org/infinispan/protostream/annotations/impl/processor/tests/ProtoSchemaTest.java

```diff
@@ -1376,6 +1379,33 @@ public void testGenericMessage() throws Exception {
       assertEquals("asdfg", ((GenericMessage.OtherMessage) genericMessage.field4.getValue()).field1);
    }
 
+   @Proto
+   static final class BareMessage {
+      public int anInt;
+      public String aString;
+      public List<String> things;
+      public Map<String, String> moreThings;
+   }
+
+   @ProtoSchema(schemaFileName = "bare_message.proto",
+         includeClasses = {
+               BareMessage.class,
+         }, syntax = ProtoSyntax.PROTO3
+   )
+   interface TestBareMessageSerializationContextInitializer extends GeneratedSchema {
+   }
+
+   @Test
+   public void testBareMessage() {
+      SerializationContext ctx = ProtobufUtil.newSerializationContext();
+
+      GeneratedSchema generatedSchema = new TestBareMessageSerializationContextInitializerImpl();
+      generatedSchema.registerSchema(ctx);
+      generatedSchema.registerMarshallers(ctx);
+
+      assertTrue(generatedSchema.getProtoFile().contains("message BareMessage"));
```

Comment:
Should we also assert the generated fields contain the correct types?

Reply:
Done